client/rest-high-level/src/main/java/org/elasticsearch/client/RollupClient.java

```diff
@@ -73,4 +75,35 @@ public void putRollupJobAsync(PutRollupJobRequest request, RequestOptions option
             PutRollupJobResponse::fromXContent,
             listener, Collections.emptySet());
     }
+
+    /**
+     * Get the Rollup Capabilities of a target (non-rollup) index or pattern
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/master/rollup-get-rollup-caps.html">
+     * the docs</a> for more.
+     * @param request the request
+     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     * @return the response
+     * @throws IOException in case there is a problem sending the request or parsing back the response
+     */
+    public GetRollupCapsResponse getRollupCapabilities(GetRollupCapsRequest request, RequestOptions options) throws IOException {
+        return restHighLevelClient.performRequestAndParseEntity(request,
+            RollupRequestConverters::getRollupCaps,
+            options,
+            GetRollupCapsResponse::fromXContent,
+            Collections.emptySet());
```

Comment:
any reason these two methods have drastically different indentation? one of them is a newline every param and the Async below is just 2 lines.